recipes-ni/cgroups-v2/nicreatecgroupsv2_1.0.bb

```diff
@@ -0,0 +1,79 @@
+SUMMARY = "NI cgroup v2 setup (mount/chown, cpusets, cpuacctgroups)"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+SRC_URI = "file://cgroupv2-common.sh \
+           file://createcpusets.sh \
+           file://createcpuacctgroups.sh \
+           file://nicreatecpusets \
+           file://nicreatecpuacctgroups \
+           file://nicreatecpusets.service \
+           file://nicreatecpuacctgroups.service \
+          "
+
+S = "${WORKDIR}"
+
+INHERIT += " systemd update-rc.d"
+
+SYSTEMD_AUTO_ENABLE = "enable"
+SYSTEMD_SERVICE:${PN} = "nicreatecpuacctgroups.service nicreatecpusets.service"
+
+INITSCRIPT_PACKAGES = "${PN}-cpusets ${PN}-cpuacct"
+INITSCRIPT_NAME:${PN}-cpusets = "nicreatecpusets"
+INITSCRIPT_PARAMS:${PN}-cpusets = "start 90 1 4 5 . stop 90 0 6 ."
+INITSCRIPT_NAME:${PN}-cpuacct = "nicreatecpuacctgroups"
+INITSCRIPT_PARAMS:${PN}-cpuacct = "start 91 2 4 5 . stop 91 0 6 ."
+
+FILES:${PN} += " /usr/lib/nicreatecgroupsv2/cgroupv2-common.sh \
+                  /usr/lib/nicreatecgroupsv2/createcpusets.sh \
```

Comment:
After we drop the cpuacct script as noted above we will end up with a single script which I think we should just install in `/usr/bin`. We can name it something more descriptive to show it's a NI script or just call it `nicreatecgroupsv2`